Change [abstract] attribute styling in Couchbase Documentation

sarahlwelton opened the pull request:
Modified CSS for .doc .quoteblock and .doc .abstract blockquote

Trialed some changes for .doc .abstract blockquote::before pseudo element, but ultimately commented them out.

There for future reference.

Images for reference of the change: 

![image](https://user-images.githubusercontent.com/110928505/220764780-d069f983-b0b4-45bb-90c9-877e2de71f42.png)
(before)

![image](https://user-images.githubusercontent.com/110928505/220764597-4451b6be-eebc-4a94-93fb-b34ff5611cf9.png)
(after)

sarahlwelton committed:
Modified CSS for .doc .quoteblock and .doc .abstract blockquote

Trialed some changes for .doc .abstract blockquote::before pseudo element, but ultimately commented them out.

There for future reference.

osfameron commented on src/css/doc.css:
```diff
@@ -494,15 +494,16 @@ ul ul ul {
 
 .doc .abstract blockquote {
-  font-size: 0.9375rem;
+  /* font-size: 0.9375rem; */
```
can just be deleted if we go ahead with this change?
e.g. we have version control, let's keep the existing CSS clean.

sarahlwelton replied:
Yeah, fair enough. I saw other commented lines in our CSS so I really wasn't sure what our approach was. (There are a lot of comments)

osfameron commented on src/css/doc.css:
```diff
@@ -494,15 +494,16 @@ ul ul ul {
-  border-left: 2px solid var(--color-brand-gray5);
+  /* font-weight: var(--weight-light);
+  border-left: 2px solid var(--color-brand-gray5); */
   padding-left: 1.125rem;
```
Looking again at the mockup, I wonder if we could remove the padding, and have this aligned like a normal paragraph?

simon-dew replied:
+1

osfameron commented on src/css/doc.css:
```diff
@@ -512,11 +513,15 @@ ul ul ul {
-  content: "Summary: ";
+/* 
+ .doc .abstract blockquote::before {
+  content: "Quick Summary \a";
```
I was trying to see where "Quick Summary" showed up... OK, it's commented out.
I'm happy for this to be deleted, but I take the point that we're not CSS experts, and so having a previously working-ish example of a `::before` pseudo-element is worth something.

Might be worth *adding* a comment describing what this block could be used for, e.g. reasons to uncomment it in future?

sarahlwelton replied:
I can do this thing.

osfameron reviewed (approved):
Looks good! Some comments inline, not urgent.

simon-dew reviewed (changes requested):
I think if we're going to remove the left border from the abstract we should remove the padding also, and just make it look like a normal paragraph. The padding by itself looks odd to me.

sarahlwelton committed:
Remove padding from abstract block...

But leave the top and bottom space?

Added comment to explain pseudo before element

simon-dew reviewed (approved):
Approved 👍 I want to try the suggested top and bottom margins for a bit and see how they feel.

sarahlwelton commented:
> Approved 👍 I want to try the suggested top and bottom margins for a bit and see how they feel.

Happy to revisit this if we decide we don't like the look, but for now, I'd just love to get this rolled out :)
